### ISSUE TITLE
Do we separate the Treasure Map?  And randomize Policy ids?

### ISSUE THREAD

**Issue by jMyles** [nucypher/nucypher-kms#63]
@michwill had an idea today of figuring out a way to distribute the Treasure Map via a separate channel form the Alice-Ursula exchange.  This way, `Policy` ids don't need to be reconstructed by Bob and can instead be completely random.

It's interesting, but I'm not sure how to put the pieces together.  At least not in the tired state I'm in now.  

Where does our thinking move next on this?

**Comment by michwill** [comment]
My thinking is this:
```
policy_group_id = hash(pub_alice, pub_bob, subpath)
policy_ids = [secure_random() for _ in range(n)]
```
`policy_ids` are just parts of the treasure map signed by Alice. `policy_group_id` can be stored with the same, staking nodes (so you can't spin up many node ids quickly), however saved in kademlia's key-value store. Kademlia's kv store is replicated, as it usually is.

Another thing which we store in kademlia kv store is correspondence node_id -> ip address, that probably should be discovered. No symlinks necessary at all in this model.

**Comment by jMyles** [comment]
Closed in #72.
